### PR TITLE
update instruction for china android [TENJIN-9827]

### PR DESCRIPTION
**Pull request by makototaguchi1110**
**JIRA**:

- https://adromance.atlassian.net/browse/TENJIN-9827

**Proposed Changes:**

- Description

**Steps:**

- [ ] Reviewed
- [ ] Deployed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,11 +120,10 @@ The Unity SDK for Tenjin. To learn more about Tenjin and our product offering, p
 
 ## <a id="oaid"></a>OAID and other Android App Stores
 
-Tenjin supports promoting on other Android App Stores using the Android OAID. We have the following options for integrating OAID libraries. **If you plan to develop app outside of Google Play, make sure to implement these OAID libraries.**
+Tenjin supports promoting on other Android App Stores using the Android OAID. We have the following requirements for integrating OAID libraries. **If you plan to develop app outside of Google Play, make sure to implement these OAID libraries.**
```
Comment (amirmanji) [inline README.md:123]: Same recommendation as the other PR:

"Tenjin supports promoting your app on other Android App Stores using the Android OAID. We have the following requirements for integrating OAID libraries. **If you plan to release your app outside of Google Play, make sure to implement these OAID libraries.**"

File: README.md [hunk 2]
```diff
@@ -333,6 +331,8 @@ To specify Tenjin as the destination for your [SK Ad Network postbacks](https://
 
 These steps are adapted from Apple's instructions at [https://developer.apple.com/documentation/storekit/skadnetwork/configuring_an_advertised_app](https://developer.apple.com/documentation/storekit/skadnetwork/configuring_an_advertised_app).
 
+**NOTE**: If you are using AppLovin MAX for mediation, their SDK will overwrite Tenjin URL with their URL during the app build process. In that case, AppLovin can forward us the postbacks, so please contact your account manager to enable the forwarding.
```
Comment (amirmanji) [inline README.md:334]: Small clean up:

"If you are using AppLovin MAX for mediation, their Unity SDK will overwrite any value you entered for `NSAdvertisingAttributionReportEndpoint` with their own URL during the build process. You should be able to set the `NSAdvertisingAttributionReportEndpoint` to `https://tenjin-skan.com` in XCode after its been overwritten, or you can ask your AppLovin account manager set up forwarding the postbacks to us."

**Review by amirmanji — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (makototaguchi1110) [reply README.md]: Thanks, updated.

Comment (makototaguchi1110) [reply README.md]: @amirmanji do you mind approving this?

**Review by hc5duke — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (makototaguchi1110) [reply README.md]: updated

**Review by amirmanji — APPROVED** [review]

**Pull request merged**